### ISSUE TITLE
Rendering issue with text from drafts in the economy tab

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#672]
It's not possible to take a screenshot of the exact issue as it's only visible when hovering the line from a draft in the economy tab.
The involved text is this one:
![draft_issue](https://user-images.githubusercontent.com/709500/67671889-06adf900-f977-11e9-8a46-593a1d0ebfce.PNG)

On the computer I noticed the issue, using a 1280x1024 screen, once hovering the line, `Pool:` move to the start of the second line, therefore pushing `Inventory` on a third line, over the next economy line.

**Comment by Lego6245** [comment]
Yeah, I've been noticing this too. I think the problem is with the recent changes to the economy logging,  the new events have very long names, so they reflow onto a second line. We should either use shorter strings or abbreviate.

**Comment by lusbenjamin** [comment]
This works properly for me in v2.12.6:
![image](https://user-images.githubusercontent.com/14894693/67696343-ec1e5480-f963-11e9-8546-204e31a855eb.png)

You _should_ also be able to filter to "Event Card Pool" and see all your new and old draft pool transactions in one place:
![image](https://user-images.githubusercontent.com/14894693/67696455-238d0100-f964-11e9-98e8-39a0a8b785e3.png)

**Comment by Porkepix** [comment]
The involved computer is still on 2.12.5 due to #671 ; I didn't do a manual update to help if anything is needed for this issue. I'll check after this on 2.12.6 if issue is still here then.
